src/com/amazon/kinesis/streaming/agent/processing/processors/AddMetadataConverter.java

```diff
@@ -58,11 +58,16 @@ public class AddMetadataConverter implements IDataConverter {
     private Object metadata;
     private Boolean timestamp;
     private final IJSONPrinter jsonProducer;
+    private boolean trackedFilePath;
```

Comment:
```suggestion
    private boolean shouldOverwriteTrackedFilePath;
```

This way it's clear what the meaning of this is.